src/main/java/com/javadiscord/jdi/internal/models/auditlog/AuditLogEntry.java

```diff
@@ -0,0 +1,17 @@
+package com.javadiscord.jdi.internal.models.auditlog;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+// missing auditLogChanges
```

Comment:
Can you explain this comments?

Reply:
We discussed on disc and decided to leave for tech debt